cot/src/admin.rs

```diff
@@ -135,6 +136,32 @@ async fn authenticate(request: &mut Request, login_form: LoginForm) -> cot::Resu
     }
 }
 
+/// Struct representing the pagination of objects.
+#[derive(Debug, Clone, Copy)]
+pub struct Pagination {
+    limit: usize,
+    offset: usize,
+}
+
+impl Pagination {
+    fn new(limit: usize, page: usize) -> Self {
+        Self {
+            limit,
+            offset: (page - 1) * limit,
```

Comment:
This will overflow when `page == 0`, causing a panic in debug mode and possibly unexpected behavior in release mode. We should do an `assert!(page > 0)` here and make sure `view_model` never passes `page=0` (for instance, by throwing 404, as I suggested).